modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -6,8 +6,14 @@
 [id='modifying-install-config-for-dual-stack-network_{context}']
 = Optional: Deploying with dual-stack networking
 
-To deploy an {product-title} cluster with dual-stack networking, edit the `machineNetwork`, `clusterNetwork`, and `serviceNetwork` configuration settings in the `install-config.yaml` file. Each setting must have two CIDR entries each. Ensure the first CIDR entry is the IPv4 setting and the second CIDR entry is the IPv6 setting.
+For dual-stack networking in {product-title} clusters, you can configure IPv4 or IPv6 address endpoints for nodes in the cluster. 
 
+You can also configure IPv4 or IPv6 virtual IP (VIP) address endpoints for the Ingress VIP and API VIP. This provides an interface to the cluster for applications that use IPv4 or IPv6 addresses.   
```

Comment:
This should probably be "IPv4 and IPv6".  In dual stack you can't have just an IPv6 VIP.